.github/dependabot.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: monthly
```

Comment:
I think weekly would be better. Otherwise we get a ton of prs at the end of the month.

Reply:
Other than that great job.

Reply:
it's true that we're on a webapp and we need to react to dependencies containing a Security fail.

Reply:
Yes. That is indeed quite important. But with dependabot it's a piece of cake.